applications/alphaday.md

```diff
@@ -0,0 +1,195 @@
+
+## Development Status :open_book:
+
+Product is live at: app.alphaday.com
+
+Landing Page: https://alphaday.com/ 
+Twitter: https://twitter.com/AlphadayHQ 
+Discord: https://discord.com/invite/8KSmPyT5k8 
+Github: https://github.com/AlphadayHQ 
+Linkedin: https://www.linkedin.com/company/alphaday
+Pitch Deck: https://alphaday.com/deck 
+Demo: https://www.youtube.com/watch?v=ThCd_W3rK_8 
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 2 months
+- **Full-Time Equivalent (FTE):**  2 FTE
+- **Total Costs:** 4,000 USD
+
+### Milestone 1 — Development & Deployment of the Dashboard
+
+- **Estimated duration:** 2 months
+- **FTE:**  2
+- **Costs:** 3,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | MIT |
+| 0b. | Documentation | Not applicable to Alphaday, as we have no Substrates nodes or sending functionality on the app |
+| 0c. | Testing Guide | We can provide a document as a guide to give a bit more detail on how to test a completed dashboard's functionality. It will be a reference to the points listed below (1-14) |
+| 0d. | Docker | The dashboard would be live and visible to anyone that accesses the url. As such, a Docker file would be deemed unnecessary. |
+| 0e. | Article | We will publish articles and other posts about the dashboard and the project once we deploy it. |
+| 1. | Blog | The latest Polkadot official announcements and blog post content |
+| 2. | News | A feed of all news mentions of Polkadot/Kusama from coindesk, cointelegraph and 20+ other news sources aggregated from across the internet |
+| 3. | Calendar | A calendar containing all Polkadot/Kusama events including meetups, hackathons with Polkadot/Kusama bounties, AMAs |
+| 4. | Documentation | Complete Polkadot/Kusama Document directory |  
+| 5. | Forum | Integration with Polkadot Governance Forum to show the latest and trending discussion topics |  
+| 6. | Social | Discord & Reddit Integration to show a feed of latest discussions for both Polkadot/Kusama |  
+| 7. | Misc | FAQ, Polkadot Whitepaper, DOT/KSM tokenomics information |
+| 8. | Token Price | Live DOT/KSM spot price chart |
+| 9. | Developer Resources | Developer tools, API’s, tutorials, guides, grants & bounties, documentation, and any other ‘Build’ resources that can help build with Polkadot/Kusama |  
+| l0. | Careers | Polkadot/Kusama/Web3 Careers integration to show the latest open jobs listed |  
+| 11. | Roadmap |  A widget featuring the Polkadot roadmap with all the descriptions and links included |  
+| 12. | Ecosystem | Categorized links and descriptions of products and services offered in the Polkadot/Kusama ecosystem |  
+| 13. | TVL Data | Polkadot/Kusama TVL data through Defillama integration |
+| 14. | Team | Polkadot/Kusama founders information |
+
+### Milestone 2 — Maintenance & Curation of Dashboard
+
+- **Estimated duration:** Indefinitely
```

Comment:
We won't be able to approve an unlimited estimated duration, could we maybe allocate 2 months here for now, since that's the timeline written in future plans.